### PR TITLE
fix: Add countryCode to DB details

### PR DESCRIPTION
**Pull request by mar-schmidt**

### CODE REVIEW

**Review by sasjo — CHANGES_REQUESTED** [review]
country code is an optional input so the action can only concatenate tenantName and countryCode if the code is set.

**Comment by mar-schmidt** [comment]
> country code is an optional input so the action can only concatenate tenantName and countryCode if the code is set.

Fixed.

File: txengine-deploy/src/env-config.js [hunk 1]
```diff
@@ -23,10 +23,12 @@ const parseEnvironment = (environment, projectId) => {
   return yaml.parse(environment.replace(/sm:\/\/\*\//g, `sm://${projectId}/`));
 };
 
-const defaultEnvironment = (projectId, tenantName) => ({
-  DATABASE_HOST: `sm://${projectId}/${tenantName}_postgresql_private_address`,
+const getConditionalCountryCodeString = (countryCode) => (typeof (countryCode) !== 'undefined' ? `${countryCode}_` : '');
```
Comment (sasjo) [inline txengine-deploy/src/env-config.js:26]: I think this will fail if countryCode is a blank string.

**Review by sasjo — CHANGES_REQUESTED** [review]
Can you add a test to verify output if country code is absent. Pretty sure that missing input becomes empty string so we need to make sure we handle that.

**Comment by mar-schmidt** [comment]
> Can you add a test to verify output if country code is absent. Pretty sure that missing input becomes empty string so we need to make sure we handle that.

Is this test not sufficient for absent country code?
```
test('It can handle optional country code')
```

**Review by sasjo — APPROVED** [review]

**Pull request merged**